package.json

```diff
@@ -17,12 +18,11 @@
   "scripts": {
     "coverage": "cross-env CI=true npm test -- --coverage",
     "lint": "eslint --quiet src __tests__ --ext .js,.jsx",
-    "start": "sudo nodemon lib/index.js",
+    "start": "sudo -E nodemon lib/index.js",
     "test": "react-scripts test --env=jsdom",
-    "verify": "npm run lint && cross-env CI=true npm test"
+    "verify": "npm run lint && cross-env CI=true npm test",
+    "preinstall": "yarn global add nodemon"
   },
   "license": "MIT",
-  "devDependencies": {
-    "nodemon": "^1.11.0"
-  }
+  "devDependencies": {}
```

Comment:
Is `nodemon` not needed as a devDependency because it gets installed in the container?